package.json

```diff
@@ -104,16 +104,16 @@
     "@testing-library/react": "^10.4.7",
     "@testing-library/react-hooks": "^3.4.1",
     "@testing-library/user-event": "^12.1.0",
-    "@types/fs-extra": "^9.0.1",
-    "@types/jest": "^26.0.8",
+    "@types/fs-extra": "^9.0.4",
+    "@types/jest": "^26.0.15",
     "@types/lodash.shuffle": "^4.2.6",
-    "@types/node": "^13",
+    "@types/node": "^14.14.7",
```

Comment:
I think we still have to rely on `@types/node: ^12` and can't update to `14`. Because latest Electron is still running on Node 12.x](https://www.electronjs.org/releases/stable#11.0.0)